mtdome/eie.yaml

```diff
@@ -0,0 +1,2 @@
+# Configuration for connecting to the EIE cRIO
+host: 139.229.168.194
```

Comment:
Currently there is no hostname yet. It will come in the future and then we will not need this Docker container anymore because then the EIE network will have been connected to the Rubin Observatory network so we can then use the CSC in the kubernetes cluster instead.